app/services/course_search_service.rb

```diff
@@ -91,6 +91,18 @@ def locatable_sites
     # Only running and published site statuses
     running_and_published_criteria = site_status[:status].eq(SiteStatus.statuses[:running]).and(site_status[:publish].eq(SiteStatus.publishes[:published]))
 
+    running_and_published_and_vacancy_criteria = if has_vacancies?
```

Comment:
<img width="1267" alt="Screenshot 2021-07-29 at 18 14 54" src="https://user-images.githubusercontent.com/616080/127535844-ef9415ac-a3f7-45b3-83ab-692ae9ae8b99.png">

Can we extract method here so we don't add to the complaint above?